docs/manual.md

```diff
@@ -663,27 +718,44 @@ The outcome is: Error                                             I@15:17:14.388
 ```
 
 In a somewhat obfuscated way, Apalache tells us the following. It has inferred
- that `Left` is a set of integers, that is, `FinSet[Int]`. First, it found that
+that `Left` is a set of integers, that is, `FinSet[Int]`. First, it found that
 `InSet` is a set of integers, by applying `ConstInit`. Second, as `Left = InSet`
 in `Init`, it inferred that `Left` is also a set of integers. Third, when
 applying `Next`, it processed `{}`, which is an empty set of any kind of
-objects. Hence, `{}` was assigned the type `FinSet[Unknown]`, that is, a set
-of some type. Finally, it found the expression `Left = {}`,
-and here the type checker has failed.
+objects. Hence, `{}` was assigned the type `FinSet[Unknown]`, that is, a set of
```

Comment:
@konnov or @andrey-kuprianov This is the other place where I've suggested substantive changes. Does it seem accurate? Is the suggested change to using `v <: T == v` actually useful, or is it more confusing, since so much of the example code uses `a <: b == a`? (Note that I could change the examples to use the suggested convention, if we thought it enough of an improvement to be worth while).

Reply:
yes, you changes seem very reasonable to me!

Reply:
It looks fine. Hopefully, this kind of type annotations goes away very soon :-)